app/src/main/java/com/example/habitsmasher/Habit.java

```diff
@@ -38,9 +39,9 @@ public Habit (String title, String reason, Date date, String days, boolean isPub
         _title = title;
         _reason = reason;
         _date = date;
-        _days = days;
         _habitEvents = habitEvents;
         _isPublic = isPublic;
+        _days = new DaysTracker(days).getListWithStrings();
```

Comment:
A lot of files are essentially just little changes like this to adapt to changes.